src/main/java/com/minecolonies/apiimp/initializer/ModBuildingsInitializer.java

```diff
@@ -48,7 +48,7 @@ public static void init(final RegistryEvent.Register<BuildingEntry> event)
                                  .setBuildingProducer(BuildingArchery::new)
                                  .setBuildingViewProducer(() -> BuildingArchery.View::new)
                                  .setRegistryName(new ResourceLocation(Constants.MOD_ID, ModBuildings.ARCHERY_ID))
-                                 .addBuildingModuleProducer(MinimumStockModule::new, MinimumStockModuleView::new)
+                                 .addBuildingModuleProducer(MinimumStockModule::new, () -> MinimumStockModuleView::new)
```

Comment:
Won't those double lambdas create java version issues like it did in structurize?

Reply:
Symptom for this issue is class being interface

Reply:
Ah okay, so it should be fine.

Reply:
This is used everywhere by forge to deal with this, and can not really cause an issue.